.github/workflows/helm.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Helm release'
+
+on:
+  push:
+    paths:
+      - 'helm-charts/l7mp-operator/Chart.yml'
+      - 'helm-charts/l7mp-ingress/Chart.yml'
+
+jobs:
+  release:
+    name: 'Helm release'
+    runs-on: ubuntu-latest
+    if: github.repository == 'l7mp/l7mp'
+    steps:
+      - name: 'checkout'
+        uses: actions/checkout@v2
+      - name: 'pyhton_client generate'
+        shell: bash
+        run: .python-client/generate
+      - name: 'k8s-operator build'
+        shell: bash
+        run: .k8s-operator/build
+      - name: 'build helm charts'
+        run: |
+          cd helm-charts
```

Comment:
I think, at this point, `l7mp/helm-charts/l7mp-operator/crds/crds.yml` should point to `l7mp/k8s-operator/crd.yaml`, which is a generated file and therefore it's not in the git repo.  `l7mp/k8s-operator/build` generates `crd.yaml`, but it does much more.  Maybe it's enough to call `l7mp/openapi/convert-schema -i ./crd.template.yml -o crd.yml -t k8s`, I'm not sure.

Reply:
Please see the changes in [40ee500](https://github.com/l7mp/l7mp/pull/99/commits/40ee500edcddf51e984fbea55d0cd4c255bbe4ba).